src/plcc/load_spec/validate_spec/validate_syntactic_spec/LL1/SpecGrammar.py

```diff
@@ -0,0 +1,99 @@
+from plcc.load_spec.parse_spec.parse_syntactic_spec.structs import (
+    CapturingTerminal, NonTerminal, Symbol, Terminal, SyntacticSpec, SyntacticRule
+)
+from .Grammar import Grammar
+from .LL1Wrapper import wrapLL1, LL1Wrapper
+from .errors import InvalidSymbolError, InvalidSyntacticSpecError
+
+def createSpecGrammar(syntacticSpec: SyntacticSpec):
```

Comment:
snake_casing: create_grammar and syntactic_spec (but not for the type SyntacticSpec).

Reply:
I'm going to stop making this comment, but you should apply it throughout.